components/homePage/Banner.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { ButtonLink } from "../UI";
+
+function Banner({ isMentorLoggedIn }) {
+  return (
+    <div className="tw-min-h-screen tw-w-full tw-overflow-hidden">
+      <div className="tw-min-h-screen tw-w-full tw-max-w-7xl tw-mx-auto tw-flex tw-items-center tw-px-4">
+        <div className="tw-w-full tw-grid tw-grid-cols-12">
+          <div className="tw-col-span-12 lg:tw-col-span-5">
+            <div className="tw-h-full tw-flex tw-items-center tw-mb-8 lg:tw-mb-0">
+              {/* text box */}
+              <div className="tw-mt-12 tw-text-center lg:tw-text-left tw-font-sans tw-space-y-4">
+                <h1 className="tw-font-medium tw-text-4xl lg:tw-text-5xl !tw-leading-[60px] tw-font-[Poppins]">
```

Comment:
I just checked it needs margin in mobile mode `mt-4 lg:mt-0`. Just see mobile mode.

Reply:
I already added tw-mt-12 for that.